spec/rails_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'rspec/rails'
 require 'shoulda/matchers'
 require 'cancan/matchers'
+include DeviseMasquerade::Controllers::UrlHelpers
```

Comment:
I include the URL helpers only for specs which need them.

Why are you including this helper, shouldn't including the Rails application URL routes be better?